lib/pdn/pdn.c

```diff
@@ -706,18 +706,20 @@ int pdn_dynamic_info_get(uint32_t cid, struct pdn_dynamic_info *pdn_info)
 		return -EINVAL;
 	}
 
+	cgcontrdp_at_rsp_buf = k_malloc(CONFIG_PDN_DYNAMIC_INFO_AT_BUF_SIZE);
```

Comment:
NULL check here? Or is that handled by `nrf_modem_at_cmd()`?

Reply:
It is checked in `nrf_modem_at_cmd()`, though I agree that an extra check would be useful for debugging purposes.